pkg/virt-launcher/virtwrap/manager_test.go

```diff
@@ -80,7 +80,7 @@ var _ = Describe("Manager", func() {
 			mockDomain.EXPECT().GetState().Return(libvirt.DOMAIN_SHUTDOWN, 1, nil)
 			mockDomain.EXPECT().Create().Return(nil)
 			mockDomain.EXPECT().GetXMLDesc(libvirt.DomainXMLFlags(0)).Return(string(xml), nil)
-			manager, _ := NewLibvirtDomainManager(mockConn)
+			manager, _ := NewLibvirtDomainManager(mockConn, false)
```

Comment:
Can you please add a test to show, that emulation still works? It is a fallback and we should guarantee it is still working.